assets/xml/objects/object_kamejima.xml

```diff
@@ -1,81 +1,99 @@
-        <Limb Name="object_kamejima_Standardlimb_00E674" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_0E" Offset="0xE674" />
-        <Limb Name="object_kamejima_Standardlimb_00E680" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_0F" Offset="0xE680" />
-        <Limb Name="object_kamejima_Standardlimb_00E68C" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_10" Offset="0xE68C" />
-        <Limb Name="object_kamejima_Standardlimb_00E698" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_11" Offset="0xE698" />
-        <Limb Name="object_kamejima_Standardlimb_00E6A4" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_12" Offset="0xE6A4" />
-        <Limb Name="object_kamejima_Standardlimb_00E6B0" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_13" Offset="0xE6B0" />
-        <Limb Name="object_kamejima_Standardlimb_00E6BC" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_14" Offset="0xE6BC" />
-        <Limb Name="object_kamejima_Standardlimb_00E6C8" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_15" Offset="0xE6C8" />
-        <Limb Name="object_kamejima_Standardlimb_00E6D4" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_16" Offset="0xE6D4" />
-        <Limb Name="object_kamejima_Standardlimb_00E6E0" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_17" Offset="0xE6E0" />
-        <Skeleton Name="object_kamejima_Skel_00E748" Type="Flex" LimbType="Standard" LimbNone="OBJECT_KAMEJIMA_LIMB_NONE" LimbMax="OBJECT_KAMEJIMA_LIMB_MAX" EnumName="ObjectKamejimaLimb" Offset="0xE748" />
-        <Animation Name="object_kamejima_Anim_00EF98" Offset="0xEF98" />
-        <Animation Name="object_kamejima_Anim_0100CC" Offset="0x100CC" />
-        <Animation Name="object_kamejima_Anim_010918" Offset="0x10918" />
-        <Animation Name="object_kamejima_Anim_0119D4" Offset="0x119D4" />
-        <Animation Name="object_kamejima_Anim_012260" Offset="0x12260" />
-        <Animation Name="object_kamejima_Anim_013264" Offset="0x13264" />
-        <Animation Name="object_kamejima_Anim_014E8C" Offset="0x14E8C" />
+        <DList Name="gTurtleWaveDL" Offset="0x140" />
+        <Texture Name="gTurtleWaveUnusedTex" OutName="turtle_wave_unused" Format="i8" Width="32" Height="64" Offset="0x2D8" />
+        <Texture Name="gTurtleWaveTex" OutName="turtle_wave" Format="i8" Width="64" Height="64" Offset="0xAD8" />
+        <TextureAnimation Name="gTurtleWaveTexAnim" Offset="0x1AF0" />
+
+
+        <!-- Turtle -->
+        <Collision Name="gTurtleAsleepCol" Offset="0x2328" />
+        <TextureAnimation Name="gTurtleEmptyTexAnim" Offset="0x2360"/>
+        <Collision Name="gTurtleAwakeCol" Offset="0x2470" />
+        <Collision Name="gTurtleUnusedCol" Offset="0x2E04" />
+        
+        <Animation Name="gTurtleGlitchy1Anim" Offset="0x3980" />
+        <Animation Name="gTurtleCoughAnim" Offset="0x47B8" />
+        <Animation Name="gTurtleIdleAnim" Offset="0x48B0" />
+
+        <DList Name="gTurtleAsleepDL" Offset="0x4E70" />
```

Comment:
I get where you are coming from, though don't think it is necessary to lengthen the name since it still is the form the turtle takes when he is asleep. Perhaps a comment explaining that he takes the form of a rock when he's asleep would be enough?

Reply:
Yeah, that would be totally fine with me. I just want something in the codebase that says "hey, he's a rock here, not a turtle" so that people who might be a little less familiar with the game don't get confused.